fixtures/pages-functions-wasm-app/README.md

```diff
@@ -0,0 +1,41 @@
+# ⚡️ pages-functions-wasm-app
+
+`pages-functions-wasm-app` is a test fixture that sets up a ⚡️Pages project with [Functions](https://developers.cloudflare.com/pages/platform/functions) and [`wasm` module imports](https://blog.cloudflare.com/workers-javascript-modules/)
+
+## Dev
+
+```bash
+# cd into the test fixture folder
+cd fixtures/pages-functions-with-routes-app
```

Comment:
```suggestion
cd fixtures/pages-functions-wasm-app
```

Reply:
lol...I double checked so many times, and I still missed some of these. Nice catch!